build.gradle.kts

```diff
@@ -72,10 +71,6 @@ allprojects {
         }
     }
 
-    tasks.withType<org.jetbrains.kotlin.gradle.tasks.KotlinCompile> {
```

Comment:
With this removed, can we also get rid of the following lines further up in the file?

```kotlin
val javaVersion = JavaVersion.VERSION_11
val jvmTargetVersion = JvmTarget.JVM_11
```

Reply:
Oh, it's in a later commit. Will remove it here.